docs/Plugins.rst

```diff
@@ -2939,10 +2939,11 @@ Lua API
 =======
 
 Some plugins consist solely of native libraries exposed to Lua. They are listed
-in the `lua-api` file under `lua-plugins`:
+in the `lua-api` file under `lua-plugins`::
```

Comment:
The extra colon is making this a code block instead (at least when I make this change on `develop`):

![image](https://user-images.githubusercontent.com/3719547/89856238-af27f500-db66-11ea-9b08-add7eccc0838.png)

Reply:
whoops -- that was the rendering I wanted when I wrote the example code. obviously I didn't check how it affected this block. will fix. one sec